Demo/Resources/snowmobile.json

```diff
@@ -0,0 +1,348 @@
+{
+    "market": "snowmobile",
```

Comment:
Where do you get the from? Just curious.

Reply:
From this subcategory. I based it on the URL we're calling once it's selected.

<img width="250" alt="screen shot 2019-01-21 at 14 17 07" src="https://user-images.githubusercontent.com/1901556/51476990-57a91500-1d87-11e9-846d-fb1bfff16fa4.png">

Reply:
Or did you mean the data itself?

Reply:
Yeah, the data 😄